BDArmory/Utils/ProjectileUtils.cs

```diff
@@ -104,6 +105,7 @@ public static void CalculateArmorDamage(Part hitPart, float penetrationFactor, f
             {
                 if (armorType == 1) return; //ArmorType "None"; no armor to block/reduce blast, take full damage
             }
+            if (BDArmorySettings.PAINTBALL_MODE) return; //don't damage armor if paintball mode
             float thickness = (float)hitPart.GetArmorThickness();
```

Comment:
I suspect this was why some craft were still getting damaged while in Paintball mode - spall/shrapnell damage from damaged armor